cloudformationTemplates/startChatContactAPI/README.md

```diff
@@ -1,83 +1,137 @@
+    ![Update Cloud Formation Stack](images/updateCloudFormationStack.png)
+    -   These are added as Lambda environment variables.
 
-Note: you will want to show the widget only when there is a conversation in progress. If the widget is visible when there is no conversation, you will just see a loading spinner.
+## Getting started with Pre Built Widget
 
-1. In your website's html code, import the 'amazon-connect-chat-interface.js' file from this repo.
+You can refer to the `widgetIndex.html` file in this repo to see an example of how to use the widget.
 
+1. In your website's html code, import the 'amazon-connect-chat-interface.js' file from this repo. Also import jQuery
     ```html
-    <script src="amazon-connect-chat-interface.js"></script>
+        <script src="amazon-connect-chat-interface.js"></script>
+        <script src="https://code.jquery.com/jquery-3.4.1.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
     ```
 
-2. Initialize the Chat Interface on page load. Note: you need to update this to include the root id of the div where the customer chat widget will live.
-
-    ```js
-    $(document).ready((a) => {
-      connect.ChatInterface.init({
-        containerId: 'root' // This is the id of the container where you want the widget to reside
-        });
-    });
+2. Create a `div` with id of `root`. 
+    ```html
+    <div id="root">
+    </div>
     ```
+3. Get your API Gateway URL by going to your AWS Console and going to API Gateway product
+
+    ![API Gateway](images/apiGatewayUrl.png) 
 
-3. Start the chat based on a user action. You will want to add fields for the customer name and username because those fields are used in the Lambda function that was created. Note: you need to update this to include the API Gateway endpoint that was created in the CloudFormation stack. To see examples of the success and failure handlers, refer to the [example implementation](https://github.com/amazon-connect/amazon-connect-chat-ui-examples/blob/master/cloudformationTemplates/asyncCustomerChatUX/website/index.html#L283).
+4. Paste the below code as a script section in your html file and ***Update the 4 TODO fields***
 
     ```js
-    connect.ChatInterface.initiateChat({
-      name: customerName,
-      username: username,
-      region: ${region},
-      apiGatewayEndpoint: "https://${apiId}.execute-api.${region}.amazonaws.com/Prod",
-      contactAttributes: JSON.stringify({
-        "customerName": customerName
-      }),
-      contactFlowId: "${contactFlowId}",
-      instanceId: "${instanceId}"
-    },successHandler, failureHandler)
+    <script>
+            $(document).ready((a) => {
+                connect.ChatInterface.init({
+                    containerId: 'chat' // This is the id of the container where you want the widget to reside
+                });
+            });
+
+            $(function () {
```

Comment:
Code:
why can't all of this go in the document.ready block above?
why are the variables wrapped in an if block? they could be defined outside of it so the customer would modify everything in one place?
You're asking the customer to use fields like #contactDetails but none of those exist in the customer's page you haven't told the customer how to use those or how to define them.

Reply:
you've also never loaded chatui to any of these pages... does this even work?

Reply:
I added the details of html page. It does work. Try after the new commit